src/Parser/DeathKnight/Unholy/Modules/Features/Abilities.js

```diff
@@ -194,39 +205,40 @@ class Abilities extends CoreAbilities {
         },
       },
       {
-        spell: SPELLS.DEATH_AND_DECAY,
-        category: Abilities.SPELL_CATEGORIES.OTHERS,
-        cooldown: 30,
+        spell: SPELLS.WRAITH_WALK_TALENT,
+        category: Abilities.SPELL_CATEGORIES.UTILITY,
+        enabled: combatant.hasTalent(SPELLS.WRAITH_WALK_TALENT.id),
         gcd: {
           base: 1500,
         },
       },
       {
         spell: SPELLS.DEATH_STRIKE,
-        category: Abilities.SPELL_CATEGORIES.OTHERS,
+        category: Abilities.SPELL_CATEGORIES.DEFENSIVE,
         gcd: {
           base: 1500,
         },
       },
+      // utility
       {
-        spell: SPELLS.CONTROL_UNDEAD,
+        spell: SPELLS.RAISE_DEAD,
         category: Abilities.SPELL_CATEGORIES.UTILITY,
+        cooldown: 30,
         gcd: {
           base: 1500,
         },
       },
       {
-        spell: SPELLS.RAISE_ALLY,
+        spell: SPELLS.CONTROL_UNDEAD,
         category: Abilities.SPELL_CATEGORIES.UTILITY,
-        cooldown: 600,
         gcd: {
           base: 1500,
         },
       },
       {
-        spell: SPELLS.WRAITH_WALK_TALENT,
+        spell: SPELLS.RAISE_ALLY,
         category: Abilities.SPELL_CATEGORIES.UTILITY,
-        enabled: combatant.hasTalent(SPELLS.WRAITH_WALK_TALENT.id),
+        cooldown: 600,
```

Comment:
I would suggest removing the cooldown since you are not limited by the normal cooldown in a raid setting.

Reply:
Good call, I'll add the change to whatever PR I do next.